<?xml version="1.0"?>
<launch>
    <node pkg="follow" type="follow_node" name="follow_node" output="screen">
        <param name="yuzhi_anquan" value="5.0"/>
        <param name="debug_on" value="1"/>
        <param name="xishu_linearx" value="0.5"/>
        <param name="xishu_angularz" value="2"/>
        <param name="yuzhi_x" value="4"/>
        <param name="yuzhi_z" value="3"/>
    </node>
</launch>
